### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by Stalexus**
hw2

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+button {
+  display:block;
+  cursor: pointer;
+  color: #FFF;
+  border: none;
+  width: 211px;
+  height: 60px;
+  border-radius: 10px;
+  font-size: 700 17px 'DM Sans', Arial, sans-serif;position: absolute;
```
Comment (shuliakleonid) [inline css/style.css:25]: style code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+}
+img {
+  width: 100%;
+}
+body {
+  color: #696871;
```
Comment (shuliakleonid) [inline css/style.css]: I suggest you write body at the top

File: index.html [hunk 3]
```diff
@@ -4,9 +4,59 @@
...
+                          <li><a href="#">Home</a></li>
+                          <li><a href="#">Feautures</a></li>
+                          <li><a href="#">Pricing</a></li>
+                          <li><a href="#">Blog</a></li>
+                        </ul>
+                     </nav>
+                                    <div>
+                                        <buttom class="up_but">Get Started</buttom>
+                                    </div>
+                            </div>
```
Comment (shuliakleonid) [inline index.html]: Please, before you push the code start a formatting

File: index.html [hunk 4]
```diff
@@ -4,9 +4,59 @@
...
+    <title>HomeWork2</title>
 </head>
 <body>
-    
+    <header class="header_up">
+        <div class="container">
+            <div class="header">
+                <div class="logo">
+                    <img src="img/Logo Color.svg">
+                </div>
```
Comment (shuliakleonid) [inline index.html:20]: add alt at img

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

**Comment by shuliakleonid** [comment]
## Проверь Свой макет
image

File: css/style.css (hunk 1 above)
Comment (Stalexus) [reply css/style.css:25]: changes applied

**Pull request merged**